conpot/emulators/proxy.py

```diff
@@ -135,7 +135,7 @@ def handle(self, sock, address):
                 except _socket.error as socket_err:
                     data = []
                     socket_close_reason = str(socket_err)
-                if len(data) is 0:
+                if len(data) != 0:
```

Comment:
This inadvertently reverses the meaning and makes the tests fail (yay for tests :smile:).

I think the original intent was to test for an empty list, which according to PEP 8 should be:

```suggestion
                if not data:
```

Reply:
thanks for catching that 🤦